website/home.py

```diff
@@ -31,16 +44,95 @@
     """
 )
 
+# Custom CSS for select box background color
+st.markdown(
+    """
+    <style>
+    .st-multiselect {
+        background-color: rgba(151, 166, 195, 0.25) !important;
+        color: black; /* Text color */
+    }
+    .st-multiselect select {
+        background-color: rgba(151, 166, 195, 0.25) !important;
+    }
+    .st-multiselect select option {
+        background-color: rgba(151, 166, 195, 0.25) !important;
+    }
+    </style>
+    """,
+    unsafe_allow_html=True,
+)
+
+# Create a sidebar for filters
+with st.sidebar:
+    st.markdown("### Filters")
+
+    # Add select boxes for filtering metadata with default values set to all items
+    selected_model_name = st.multiselect(
+        "Select Model Name",
```

Comment:
```suggestion
        "Model Name",
```
And similarly for the ones below